packages/design-system/src/components/Dialog/Dialog.tsx

```diff
@@ -335,20 +178,17 @@ export const Dialog = (props: DialogProps) => {
         </header>
         <main role="main" className="ds-c-dialog__body">
           <div id="dialog-content">{children}</div>
-
           {actions && <div className={actionsClassNames}>{actions}</div>}
         </main>
       </div>
-    </AriaModal>
+    </NativeDialog>
   );
 };
 
 Dialog.defaultProps = {
   closeButtonVariation: 'ghost',
   closeIcon: <CloseIcon />,
-  escapeExits: true,
-  escapeExitDisabled: false,
-  underlayClickExits: false,
+  scrollDisabled: true,
```

Comment:
We should remove this prop and bake the functionality into this component.

If a modal is open, there's no reason an application would want to allow users to continue to scroll the background/body content. 

A true modal should prevent interaction with non-modal things, and the dark backdrop that appears when the modal is open obscures any readable text anyway.

Reply:
That seems totally reasonable to me 👍